debian/jessie/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM debian:jessie
+
+LABEL maintainer me@danvaida.com
+
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update -y && \
+    apt-get install --fix-missing && \
```

Comment:
unfortunately, it can happen sometimes that you want to install some package and this package will need to update some other package(s). The typical, nasty dependency hell/waterfall. Given this is intended to be an automated build, I can't intervene to solve conflicts/issues.